app/templates/Membership_Application_Cancellation_Confirmation.twig

```diff
@@ -0,0 +1,57 @@
+<html>
+<section>
+	<div class="container clearfix">
+		<div class="ltcol">
+			&nbsp;
+		</div>
+
+		<div class="ctcol">
+			<div class="box box-common partial rounded no-margin">
+				<div class="box-header container clearfix">
+					{% if cancellationSuccessful %}
+					<span class="icon-ok">
+						{$ 'membership-cancellation-header'|trans({}) $}
```

Comment:
If you don't have tranlsation parameters, you can use just `|trans` instead of `|trans({})`.